lib/Rapid.js

```diff
@@ -3,10 +3,11 @@ const isFunction = require('lodash/isFunction');
 const flatten = require('lodash/flatten');
 const identity = require('lodash/identity');
 const camelCase = require('lodash/camelCase');
-const uuid = require('uuid/v4');
+const { v4: uuidv4 } = require('uuid');
```

Comment:
Could all of these be replaced by `node:crypto`?

Reply:
Using `randomUUID()` right?